telegram/_bot.py

```diff
@@ -3800,17 +3829,20 @@ async def get_chat_administrators(
         connect_timeout: ODVInput[float] = DEFAULT_NONE,
         pool_timeout: ODVInput[float] = DEFAULT_NONE,
         api_kwargs: JSONDict = None,
-    ) -> List[ChatMember]:
+    ) -> Tuple[ChatMember, ...]:
         """
         Use this method to get a list of administrators in a chat.
```

Comment:
Yes, you're right, I didn't update the docstrings. We could convert every mention of list to array. I'm not sure if that's worth the effort … I think I vote +0. let's see what others think :)

Reply:
I'm definitely not sure it's worth the effort, but yes, it would make the docstrings consistent with Telegram API. The problem is that "Array" can mean a mutable thing too.